include/swift/AST/DiagnosticsSema.def

```diff
@@ -7131,8 +7133,8 @@ ERROR(property_wrapper_computed, none,
       "property wrapper cannot be applied to a computed property",
       ())
 ERROR(property_wrapper_effectful,none,
-    "property wrappers currently cannot define an 'async' or 'throws' accessor",
-    ())
+      "property wrapper 'wrappedValue' property cannot currently define an 'async' or 'throws' %0",
+    (DescriptiveDeclKind))
```

Comment:
personally i agree with you about this, because "yet" and "currently" doesn't imply that in future it's going to be supported. 

done in: 4d2c617976d